paper.md

```diff
@@ -2,11 +2,88 @@
 title: "Popper 2.0: A Container-Native Workflow Execution Engine For Testing Complex Applications and Reproducing Scientific Explorations"
 author: Ivo Jimenez, Jayjeet Chakraborty, Arshul Mansoori, Quincy Wofford and Carlos Maltzahn
 abstract: |
- Abstract
+    Reproducibility and Replication crisis is still a prevalent problem
+    in scientific research. Researchers exploring the work done
+    in a field often find it difficult reproducing the experiments from their
+    artifacts, i-e the code, data, diagrams, results left behind by the previous 
+    researcher. Code developed on one machine often fail to run in other
+    machines due to differences in OS, architecture, etc. This is accompanied
+    by the difficulty of organising and using the experimental artifacts in correct 
+    order. People have tried to solve this problem in the past and has been successfull
+    to an extent by building and using workflow execution engines. With the advent of 
+    light-wight virtualization technologies in the form of containers, people 
+    have built various workflow engines for executing workflows inside 
+    containers through Kubernetes clusters. Some workflow execution engines do not
```

Comment:
@ivotron Can you please give an example here ?

Reply:
"We can divide workflow engines in two categories, those that enforce the use of containers (Argo, Brigade, etc.), and those that don't (Pegasus, Airflow, etc.). In this second category, workflows are less portable since they depend on...."